The-SMIRNOFF-force-field-format.md

```diff
@@ -134,12 +217,12 @@ The interaction scaling parameters applied to atoms connected by a few bonds are
 
 Currently, no child tags are used because the charge model is specified via different means (currently library charges or BCCs).
 
-**QUESTION:** Should we also also require the `cutoff` (or lack thereof) to be specified?
-
-**QUESTION:** Should we also also require the `switch` (or lack thereof) to be specified, since this can also have a large effect on the forces near the cutoff?
+For methods where the cutoff is not simply an implementation detail but determines the potential energy of the system (`reaction-field` and `Coulomb`), the `cutoff` distance must also be specified, and a `switch-width` if a switching function is to be used.
 
 **TODO:** Also add the dispersion correction behavior to `StericsForce`.
 
+**QUESTION:** I've opted to not make `<LibraryCharges>` and `<BondChargeCorrections>` be sub-tags of `<ElectrostaticsForce>` since, while this may logically make sense, it makes the code *much* harder to make truly modular. For example, adding a polarizable force would require we modify the original code inside the `openforcefield` toolkit, rather than simply create a new experimental `ForceGenerator` outside the toolkit and import it for experimentation. Is this OK, or will it lead to logical problems?
```

Comment:
This sounds great to me. Modularity is much more important than "all the tags seem to be in the ideal logical order". And the stretch to have them NOT be sub-tags is not that big.

Reply:
I agree with David